pkgs/tools/networking/curl/7.15.nix

```diff
@@ -34,7 +36,7 @@ stdenv.mkDerivation rec {
   '';
 
   configureFlags = [
-      "--with-ca-bundle=/etc/ssl/certs/ca-certificates.crt"
+      "--with-ca-bundle=/foobar/ca-certificates.crt"
```

Comment:
This reverts a change introduced by @edolstra following the PR #12748. As you are aware of #8247, how does it fit in the big picture ?
Why not keep this line as is, as you patch already fixes the issue ?